### DISCUSSION TITLE
The fuck is this DMCA bruh

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [nillerusr/source-engine#345] [General]
**Are you the copyright holder or authorized to act on the copyright owner's behalf?**

Yes, I am authorized to act on the copyright owner's behalf.

**Are you submitting a revised DMCA notice after GitHub Trust & Safety requested you make changes to your original notice?**

Yes

**Please provide the Zendesk ticket number of your previously submitted notice. Zendesk ticket numbers are 7 digit ID numbers located in the subject line or body of your confirmation email.**

2604262

**Does your claim involve content on GitHub or npm.js?**

GitHub

**Please describe the nature of your copyright ownership or authorization to act on the owner's behalf.**

I am the [private] of the tyabus/xash3d repository, and authorized to act on [private]'s behalf on this matter.

**Please provide a detailed description of the original copyrighted work that has allegedly been infringed. If possible, include a URL to where it is posted online.**

[private]  original copyrighted work, which is currently being violated, is located at:  https://github.com/tyabus/xash3d/blob/master/engine/platform/posix/crashhandler_libunwind.c
Infringing material which was based on on the [private] work: https://github.com/nillerusr/source-engine/blob/master/launcher/android/crashhandler.cpp

**What files should be taken down? Please provide URLs for each file, or if the entire repository, the repository’s URL.**

https://github.com/nillerusr/source-engine/blob/master/launcher/android/crashhandler.cpp

**Do you claim to have any technological measures in place to control access to your copyrighted content? Please see our <a href="https://docs.github.com/articles/guide-to-submitting-a-dmca-takedown-notice#complaints-about-anti-circumvention-technology">Complaints about Anti-Circumvention Technology</a> if you are unsure.**

No

**<a href="https://docs.github.com/articles/dmca-takedown-policy#b-what-about-forks-or-whats-a-fork">Have you searched for any forks</a> of the allegedly infringing files or repositories? Each fork is a distinct repository and must be identified separately if you believe it is infringing and wish to have it taken down.**

Based on the representative number of forks I have reviewed, I believe that all or most of the forks are infringing to the same extent as the parent repository.

**Is the work licensed under an open source license?**

Yes

**Which license?**

GNU General Public License v3.0

**How do you believe the license is being violated?**

The infringing repository contains proprietary code of the [private], whose license terms are: [private]

If i mention that it's leaked code that means it's not present in the officially released SDK by [private].  
Official SDK can be found here: [private]

Here are some examples:  
[private]  
[private]  
[private]  
[private]  
[private]  
[private]  

The initial commit is from a repository ([private]) which have been taken down by the [private], here's the link on your public DMCA database: [private]

**What changes can be made to bring the project into compliance with the license? For example, adding attribution, adding a license, making the repository private.**

Exclude all the proprietary code or remove the repository.

**What would be the best solution for the alleged infringement?**

Reported content must be removed

**Do you have the alleged infringer’s contact information? If so, please provide it.**

No, i don't know

**I have a good faith belief that use of the copyrighted materials described above on the infringing web pages is not authorized by the copyright owner, or its agent, or the law.**

**I have taken <a href="https://www.lumendatabase.org/topics/22">fair use</a> into consideration.**

**I swear, under penalty of perjury, that the information in this notification is accurate and that I am the copyright owner, or am authorized to act on behalf of the owner, of an exclusive right that is allegedly infringed.**

**I have read and understand GitHub's <a href="https://docs.github.com/articles/guide-to-submitting-a-dmca-takedown-notice/">Guide to Submitting a DMCA Takedown Notice</a>.**

**So that we can get back to you, please provide either your telephone number or physical address.**

Phone number: [private]  
E-Mail: [private]  

**Please type your full legal name below to sign this request.**

[private]

**Comment by stevefan1999-personal** [comment]
How come does this pussy @tyabus have the audacity to speak on behalf of Xash3D?
![image](https://github.com/nillerusr/source-engine/assets/29133953/eb71fa4f-a087-4e89-8743-ca22fae15ef9)

**Comment by tyabus** [comment]
Hello there!

I submitted the DMCA request on behalf of the Xash3D-NG fork, not xash3d as
a whole.
We own the copyright rights on this code, if you paid attention (no
offense), and there is no crash handler code in the original that uses
unwind.
The DMCA can possibly be resolved in a private matter, e-mail me at my
commit email.

Thanks.

**Reply by Aleph-202** [reply]
Копа фиш порвался) Какой же ты жалкий

**Reply by stevefan1999-personal** [reply]
> We own the copyright rights on this code, if you paid attention (no offense), and there is no crash handler code in the original that uses unwind.

Look, buddy, you are making a clown out of yourself with that statement. I'm very much offended this. Anyone can use libunwind to solve the DWARF debug information to get context and instruction pointer. It is a common library specifically for exception handling. If what you say is true, anyone who used libunwind can be DMCA'd by you.

Just because something is a coincidence doesn't mean it is justifiable for a DMCA takedown request. Heck I even used libunwind in my own kernel written in Rust.

If you still cannot give any substantial evidence that repo contained something that is violating DMCA, I will be filing a countersuit soon. Or I will contact Valve for a final judgement.

**Reply by stevefan1999-personal** [reply]
Plus, the continuation of Xash3D FWGS also used unwind: https://github.com/FWGS/xash3d-fwgs/blob/1677835b45d8c3bae3a53fda2838de4c38a2da0f/engine/common/crashhandler.c#L357

Why are you not filing a DMCA for them? 

I'll be honest with you, they have more permission than you, to speak out for DMCA takedown request.

**Reply by lminos** [reply]
tyabus, 
Pathetically, looks like you screwed up. 
There are a lot of people here and this community could be against you, offended or not.
They're on any social and chat media that links that repository.

Arguments without justified evidence to get a DMCA is absurdly pointless. 
The leak is more than 7 years old and it has been improved over time under the shadows without commercial use/global advertisement. Just experimental reasons. There have been other leaks from other times (2003, 2007, 2011, ...).

It's already known that the leaked engine source code can't be used commercially, without the consent of the author,  if a product is developed from that and manipulating it to your liking, even if the source code is public, doesn't have a public license like MIT and it isn't allowed to change the license. 

Anyway, the leak won't take down nor stop. This won't change anything.
There are thousands of copies, even coming from the original ones, other repositories and hundreds of forks.

**Reply by GreMirid** [reply]
![image](https://github.com/nillerusr/source-engine/assets/56092546/31735f56-6128-4d0f-af51-44d4b7cf3b90)
Казах какой то странный, на уровне так сказать.

**Reply by schm1dtmac** [reply]
Looks like your DMCA got reversed and nullified @tyabus, so clearly you were lying the whole time.
